### PR TITLE
KevS-SK1480-Creation-Manual-installation-Grav-on-WebHosting-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
Translations still in progress.

DE, EN, ES are already done.

### CODE REVIEW

**Comment by Kazarus4** [comment]
All translations are done.

Need proof by @tcpdumpfbacke for DE and EN subs and by @Y0Coss  for latin and PL subs.

DO NOT MERGE AFTER THAT BECAUSE I MUST UPDATE ANOTHER GUIDE IN THE SAME TIME.

File: pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-es.md
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-es.md]:
```suggestion
**Descubra cómo instalar manualmente el CMS Grav en un alojamiento web de OVHcloud.**
```

File: pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-us.md
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-us.md]:
```suggestion
**Descubra cómo instalar manualmente el CMS Grav en un alojamiento web de OVHcloud.**
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by Kazarus4** [comment]
DO NOT MERGE WHILE https://github.com/ovh/docs/pull/6042 NOT READY TO MERGE, SPIP AND TYPO WILL NOT TRANSLATED TOO

**Review by bs-ovh — APPROVED** [review]

**Comment by Kazarus4** [comment]
READY to MERGE when #6050 will be proof for DE and EN, and EN corrections duplicated.

**Pull request merged**
